my_list.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'module'
+
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+end
```

Comment:
- [ ] The MyList class should contain your own functionality of the Ruby `#each` method. And the enumerables in the module.rb file should use this implementation. Kindly add it here.

  ```
  def my_each
    return to_enum(:each) unless block_given?

    counter = 0
    while counter < @list.length
      yield(@list[counter])
      counter += 1
    end
    @list
  end
  ```